apps/hash-api/src/graph/index.ts

```diff
@@ -47,6 +48,52 @@ export const zeroedGraphResolveDepths: GraphResolveDepths = {
   hasRightEntity: { incoming: 0, outgoing: 0 },
 };
 
+export const currentTemporalAxes: QueryTemporalAxesUnresolved = {
```

Comment:
Maybe call this `currentTimeInstantTemporalAxes` or something?

Reply:
I was struggling with the name. I think yours is at least better 😄

Reply:
[Rename currentTemporalAxes to currentTimeInstantTemporalAxes](https://github.com/hashintel/hash/pull/2542/commits/22828cd1ad004368546fa11b86c177cc20ea3271)